eif_defs/src/lib.rs

```diff
@@ -341,7 +381,7 @@ mod tests {
     #[test]
     fn test_eif_section_header_from_be_bytes_invalid_section_type() {
         let mut bytes = [0u8; 12];
-        bytes[1] = 5;
+        bytes[1] = 6;
```

Comment:
We could add a small comment explaining the logic, as to why this is invalid.

Reply:
Sure, added.